Gemfile

```diff
@@ -1,2 +1,6 @@
 source "https://rubygems.org"
-gemspec
\ No newline at end of file
+gemspec
+
+group :development do
+  gem 'rspec'
```

Comment:
I think it's better to add this as a development dependency [here](https://github.com/alphasights/cxml/blob/master/cxml.gemspec).